clarity/contracts/arkadiko-collateral-types-v1-1.clar

```diff
@@ -112,6 +112,7 @@
   )
 )
 
+;; TODO - this is not in use
```

Comment:
Can we remove this method? If governance wants to update the collateral types, an updated contract can be pushed I guess? 

Or we could keep this method for now and make sure the dao-owner can call this so we can quickly update if needed.

Reply:
@nieldeckx what should we do? there's also the `change-risk-parameters` method that can be called by the DAO keys...